### PR TITLE
Update CHANGELOG for v3.11.0

### PR DESCRIPTION
**Pull request by Jaifroid**
Please check that everything is included and clearly expressed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,27 @@ Please note that this application has changed its name over time.
...
 
+## Kiwix JS v3.11.0
+
+Released on *TODO*
+
+* FEATURE: Integration of in-app ZIM Download Library based on library.kiwix.org
+* FEATURE: Fallback to basic ZIM library for browsers that don't support library.kiwix.org
+* [TODO] FEATURE: Remember picked files and folders between app launches with File System Access API
+* FEATURE: Slide away header and footer when scrolling an article, restored when scrolling up
+* FEATURE: On non-scrollable pages, use Ctrl/Cmd + UpArrow/DownArrow, long swipe or mousewheel
```
Comment (Jaifroid) [inline CHANGELOG.md:15]: Edit to "mousewheel/touchpad"

**Comment by Jaifroid** [comment]
PS It would be a good idea to create an issue for the File System UI and/or Unit tests you mentioned. Definitely a separate issue, as I'd really like to get this merged asap, so we can do #1143.

**Comment by Jaifroid** [comment]
@Rishabhg71 Could you check this PR and if you're happy with the way the changes are described, please approve it. Could you also look at https://github.com/kiwix/kiwix-js/releases/tag/untagged-ef84b2b3e3e992cd2c1a (but don't publish it yet!) to see if it also looks OK? Those detailed notes are more or less auto-generated.

**Comment by Jaifroid** [comment]
By the way, the linux BrowserStack tests keep falling over....

**Comment by Rishabhg71** [comment]
> @Rishabhg71 Could you check this PR and if you're happy with the way the changes are described, please approve it. Could you also look at https://github.com/kiwix/kiwix-js/releases/tag/untagged-ef84b2b3e3e992cd2c1a (but don't publish it yet!) to see if it also looks OK? Those detailed notes are more or less auto-generated.

This looks good


> By the way, the linux BrowserStack tests keep falling over....

Weird as it passed on my PRs but if it continues I will check these tests again

**Pull request merged**